jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/OdeProperties.java

```diff
@@ -42,26 +41,27 @@
     private String ddsCasUsername = "";
     private String ddsCasPass = "";
     private String ddsWebsocketUrl = "wss://webapp2.connectedvcs.com/whtools23/websocket";
-    private String kafkaTopicBsmSerializedPOJO = "topic.J2735Bsm";
-    private String kafkaTopicBsmJSON = "topic.J2735BsmRawJSON";
-    
+    private String kafkaTopicBsmSerializedPojo = "topic.J2735Bsm";
+    private String kafkaTopicBsmRawJson = "j2735BsmRawJson";
+    private String kafkaTopicBsmFilteredJson = "j2735BsmFilteredJson";

```

Comment:
Yes, it should but changing it now will require changes in the PPM and it's not important enough. It's just a name. Let's just live with one little inconsistency.